src/hotspot/share/gc/shared/plab.hpp

```diff
@@ -169,12 +170,13 @@ class PLABStats : public CHeapObj<mtGC> {
   virtual size_t compute_desired_plab_sz();
 
  public:
-  PLABStats(const char* description, size_t desired_net_plab_sz_, unsigned wt) :
+  PLABStats(const char* description, size_t default_plab_sz, size_t desired_net_plab_sz_, unsigned wt) :
```

Comment:
Since you  are touching this line it would be nice to remove the trailing `_` in `desired_net_plab_sz_`.